runtime/kernel/exceptions.c

```diff
@@ -42,7 +42,7 @@ void zephirt_throw_exception(zval *object TSRMLS_DC){
 /**
  * Throws a zval object as exception
  */
-void zephirt_throw_exception_debug(zval *object, const char *file, zend_uint line TSRMLS_DC){
+void zephirt_throw_exception_debug(zval *object, const char *file, uint line TSRMLS_DC){
```

Comment:
ping @sjinks, can you help me
Should I change `zend_uint` to `uint`, or it's wrong?
Thanks